app/src/App/App.tsx

```diff
@@ -305,6 +310,14 @@ const App: React.FC = () => {
               {...planCreateProps}
               component={CreatePlanView}
             />
+            <PrivateComponent
+              redirectPath={APP_CALLBACK_URL}
+              disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+              exact
+              path={`${INVENTORY_SERVICE_POINT_PROFILE_VIEW}/:servicePointId`}
```

Comment:
would be nice, to get the servicePointId exported as named constant as opposed to using at as a magic string.

Reply:
@p-netm 
What do you suggest, where should constant be created for this, 
because I also have to use this in inventory profile.

Reply:
you can export this from the package, since thats where its identity is most used.